lib/fluent/fluent_log_event_router.rb

```diff
@@ -0,0 +1,137 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fluent/log'
+
+module Fluent
+  # DO NOT write any logic here
+  class NullFluentLogEventRouter
+    def start; end
+
+    def stop; end
+
+    def graceful_stop; end
+
+    def emit_event; end
```

Comment:
No argument. is it intended?

Reply:
Good catch!  I missed adding an argument.
https://github.com/fluent/fluentd/pull/2667/commits/0970e78b9f17b5fcccc2524f529ceba573944b90

Reply:
should add test for `NullFluentLogEventRouter` methods?

Reply:
I added https://github.com/fluent/fluentd/pull/2667/commits/65b7429c03cc7fc9eead11a506338d9859c44f5e